### PR TITLE
Documentation icons

### PR DESCRIPTION
**Pull request by kasperpeulen**
image

### CODE REVIEW

File: .idea/codeStyleSettings.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (lukechurch) [inline .idea/codeStyleSettings.xml]: I don't think we put these in the repo in general?

Comment (kasperpeulen) [inline .idea/codeStyleSettings.xml]: oops didn't meant to do that

**Comment by lukechurch** [comment]
lgtm

**Comment by kasperpeulen** [comment]
@lukechurch I was thinking about doing the header (int in this case) maybe a little bit larger, like font-size 13pt, thoughts ?

**Comment by lukechurch** [comment]
@kasperpeulen A little bigger sounds good to me I think. Then lets bring this in.

**Comment by lukechurch** [comment]
I'm going to merge this now, we can tweak going forwards

**Pull request merged**